python/sglang/srt/disaggregation/conn.py

```diff
@@ -293,16 +344,33 @@ def get_session_id(self):
 
 
 class KVSender:
-
     def __init__(self, mgr: KVManager, bootstrap_addr: str, bootstrap_room: int):
         self.kv_mgr = mgr
         self.bootstrap_room = bootstrap_room
         self.kv_mgr.set_status(bootstrap_room, KVPoll.WaitingForInput)
         self.aux_index = None
+        self.bootstrap_server_url = bootstrap_addr
+
+    @cache
+    def _connect_router(self, endpoint: str):
+        socket = zmq.Context().socket(zmq.DEALER)
+        self.identity = str(uuid.uuid4()).encode()
+        socket.setsockopt(zmq.IDENTITY, self.identity)
+        socket.connect(endpoint)
+        return socket
 
     def init(self, num_kv_indices: int, aux_index: Optional[int] = None):
         self.aux_index = aux_index
         self.num_kv_indices = num_kv_indices
+        # Send rank and tp size to bootstrap server (prefill node)
+        self._connect_router("tcp://" + self.bootstrap_server_url).send_multipart(
+            [
+                "Prefill".encode("ascii"),
+                str(self.identity).encode("ascii"),
```

Comment:
DEALER socket need not send `identity` explicitly. Related header will be added by ZMQ library.